src/redis_ai_objects/model_struct.h

```diff
@@ -32,15 +32,3 @@ typedef struct RAI_Model {
     long long datalen;
     void *infokey;
 } RAI_Model;
-
```

Comment:
Maybe this whole file is redundant, and this structs should move to `model.h`? WDYT?

Reply:
The initial idea was to have files with structs that could be cleanly included without risking circular dependencies. Probably in the end this is not needed that much, your call.

Reply:
let's discuss it and decide
different PR scope